core/src/trezor/lvglui/i18n/locales/zh_cn.py

```
@@ -820,7 +820,7 @@
     "选择帐户",
     "前往账户",
     "输入格式错误",
-    "ERC-20 代币在 Ethereum 网络上发行。这意味着可以使用 ETH 钱包地址接收任何 ERC-20 代币。",
+    "当前地址是一个 EVM 网络地址。您可同步管理其他 EVM 兼容网络（如 Ethereum、BNB Chain、Polygon、Arbitrum One、Avalanche等）上的资产。",
```

Ruff (0.8.2)

String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)